reframe/core/config.py

```diff
@@ -551,7 +552,7 @@ def handler_list(handler_config, basedir=None):
 def _find_config_file():
     # The order of elements is important, since it defines the priority
     prefixes = [
-        os.path.join(os.getlogin(), '.reframe'),
+        os.path.join(getpass.getuser(), '.reframe'),
```

Comment:
Better use the `os_ext.osuser()` which does this, but captures also the exception that can be possibly thrown and you should also check that the return value is not `None` and add also a unit test to test that the `None` is handled correcty.